### DISCUSSION TITLE
interesting problem I don't understand

### DISCUSSION THREAD

**Discussion by billytrip** [Motion-Project/motion#1671] [Q&A]
I've been using motion with 3 rtsp cameras for a few months.  All worked great.  I had a camera fail because of an issue with the solar controller I was using on a remote camera.  I replaced the controller.

The affected camera did not seem to be working after the fix.  But.... after a day or two I noticed I was getting motion videos from that camera.  If I use the "localhost:8080" on my browser, I can see the other two cameras but the affected camera shows 
"no connection".  Yet, I continue to get motion videos from this "no connection" camera.

What gives?

**Comment by Mr-Dave** [comment]
Check the log to see if it sometimes connects, then disconnects, then connects, etc.

**Reply by billytrip** [reply]
You are right, it is cycling pretty fast.  Now I just need to figure out why :)
